app/main.py

```diff
@@ -1,8 +1,29 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+
+    for i in range(len(people)):
+        person_list.append(Person(people[i]["name"], people[i]["age"]))
```

Comment:
Rewrite in one line using list comprehension

Reply:
Mind not to iterate by indexes also here

Reply:
I must try to avoid all iterations by indexes? I kinda like them, since it's easier to navigate through the iteration for me

Reply:
You can avoid them here to write less code, so you won't need to get a person by index and then name. Without indexes here you'll already have a person and then just get a name